src/io/flutter/FlutterMessages.java

```diff
@@ -29,6 +29,16 @@ public static void showError(String title, String message) {
                        NotificationType.ERROR));
   }
 
+  public static void showMessage(String title, String message) {
+    final Notification notification = new Notification(
+      FLUTTER_NOTIFICATION_GOUP_ID,
+      title,
+      message,
+      NotificationType.INFORMATION);
```

Comment:
I was thinking about that - the toast API does distinguish between info, warning, and error. `NotificationType.INFORMATION` doesn't have any proscriptive UI - it's fairly bland - so good as a catch-all, but doesn't convey something like an error state.